bokeh/server/blaze/views.py

```diff
@@ -35,8 +34,8 @@ def _make_range(r):
 @mbsbp.route("/render/<docid>/<datasourceid>/<glyphid>", methods=['GET', 'POST'])
 def render(docid, datasourceid, glyphid):
     #load bokeh document
-    bokehuser = bokeh_app.authentication.current_user()
-    request_username = bokehuser.username
+    #bokehuser = bokeh_app.authentication.current_user()
+    #request_username = bokehuser.username
     clientdoc = bokeh_app.backbone_storage.get_document(docid)
```

Comment:
Yeah, I tried to see if it was for the side-effects, but most seem just forgotten stuff. Someone should probably double-check this once this is ready for review.